static/js/rest.js

```diff
@@ -102,13 +107,19 @@ export const endpoints: Array<Endpoint> = [
   {
     name: 'payment',
     url: '/api/v0/payment/',
-    makeOptions: (total, klassId) => ({
+    fetchOptions: (params) => ({
```

Comment:
are you referring to the switch from individual named parameters to `params` (which is expected to be an object)? i did this b/c with the addition of parametrized urls, we have 2 different properties of `Endpoint` that can expect parameters (the `fetchOptions` function and the `urlTemplate`). here are example usages of the actions:

```javascript
dispatch(
  actions.payment({
    paymentAmount: paymentAmount,
    klassId: selectedKlass.klass_id
  })
)

dispatch(
  actions.klasses({username: SETTINGS.user.username})
)
```

`actions.klasses` uses `username` to apply to the url template, and `actions.payment` uses `paymentAmount` and `klassId` to pass to the `fetch...` function. just using sequential arguments and a splat operator wouldn't work for usage in those 2 different places. not sure what you mean by "the rest parameter". open to further suggestions here